src/components/ErrorBoundary/ErrorBoundary.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import { ComponentStory } from "@storybook/react";
+import { ErrorBoundary } from ".";
+import ErrorFallback from "./ErrorFallback";
+
+export default {
+  title: "Components/ErrorBoundary",
+  component: ErrorFallback,
+};
+
+export const Template: ComponentStory<typeof ErrorFallback> = () => (
+  <ErrorFallback />
+);
+
+Template.storyName = "Error Fallback";
+
+export const ErrorBoundaryTemplate: ComponentStory<
```

Comment:
I'm not sure if it's worth having this story, just because I think you'll need to add an extra script in `evergreen.yml` to upload the SVG for it to even appear on Storybook Production

Reply:
It's ok I can add the script with some copy+pasta